qa/tasks/cephfs/test_volumes.py

```diff
@@ -662,6 +665,38 @@ def test_volume_info_without_subvolumegroup(self):
                          "'pending_subvolume_deletions' should not be present in absence"
                          " of subvolumegroup")
 
+    def test_volume_info_with_human_readable_flag(self):
+        """
+        Tests the 'fs volume info --human_readable' command
+        """
+        vol_fields = ["pools", "used_size", "pending_subvolume_deletions", "mon_addrs"]
+        group = self._generate_random_group_name()
+        # create subvolumegroup
+        self._fs_cmd("subvolumegroup", "create", self.volname, group)
+        # get volume metadata
+        vol_info = json.loads(self._get_volume_info(self.volname, "--human_readable"))
+        for md in vol_fields:
+            self.assertIn(md, vol_info,
+                          f"'{md}' key not present in metadata of volume")
+        self.assertEqual(vol_info["used_size"], 0,
+                         "Size should be zero when volumes directory is empty")
+
+    def test_volume_info_with_human_readable_flag_without_subvolumegroup(self):
+        """
+        Tests the 'fs volume info --human_readable' command without subvolume group
+        """
+        vol_fields = ["pools", "mon_addrs"]
+        # get volume metadata
+        vol_info = json.loads(self._get_volume_info(self.volname, "--human_readable"))
+        for md in vol_fields:
+            self.assertIn(md, vol_info,
+                          f"'{md}' key not present in metadata of volume")
+        self.assertNotIn("used_size", vol_info,
```

Comment:
Sorry I missed this in the original PR. What's the difference between `used_size` and `used` field ? From the code, I can see that the `used_size` is showing the rstat size of the `/volumes` path. 

@vshankar @neesingh-rh 
Do we require the `used_size` since as it's inaccurate and we already are displaying `used` information w.r.t each pools from df output?
Even if decide to keep it, it gets filled in if  subvolume/subvolumegroup is created. Not specific to subvolumegroup.

Reply:
Also 'used_size' is not converted to human readable even with the flag.

Reply:
As we can see from the code, `used_size` is the rstat size of `/volumes` ,it updates as and when subvolumegroup/subvolume is created inside `/volumes` that should be fine coz we want info specific to a volume whereas `used` is specific to `pools` and `metadata_pools` .If total bytes used is not required above the `used` field which is specific to `pools`/`metadata_pools`,we can remove ,I guess.

Reply:
> Also 'used_size' is not converted to human readable even with the flag.

As discussed in this https://github.com/ceph/ceph/pull/45949#discussion_r938164088 , I did applied flag to `avail` and 'used` fields .If required, which should be, will update.

Reply:
Let's apply human readable flag to 'used_size' field as well.